# gVCA : Current vacuum value on Channel A
uint16 g_vca

# gVCB : Current vacuum value on Channel B  
uint16 g_vcb